rascal-lsp/src/main/rascal/lsp/lang/rascal/lsp/IDECheckerWrapper.rsc

```diff
@@ -116,6 +121,25 @@ private bool inWorkspace(set[loc] workspaceFolders, loc lib) {
     }
 }
 
+loc pathConfigFile(PathConfig pcfg) = pcfg.bin + "rascal.pathconfig";
```

Comment:
We will might need this file for a long time, possibly also after porting this behaviour to the compiler. So we should pick a good name once.